src/main/java/de/tum/in/www1/artemis/repository/UserRepository.java

```diff
@@ -458,4 +458,17 @@ default void removeOrganizationFromUser(Long userId, Organization organization)
             save(user);
         }
     }
+
+    /**
+     * Return true if the current users' login matches the provided login
+     * @param login user login
+     * @return true if both logins match
+     */
+    default boolean isCurrentUser(String login) {
```

Comment:
Should this be in the Repository? It doesn't do anything related to data storage. I think this is better stored in a Service.

Reply:
As requested by Stephan this should be in the repository.